parsers/CL_SIC.py

```diff
@@ -210,36 +211,66 @@
   "HBS GNL": "gas",
   "Rey": "oil",
   "El Nogal": "oil",
-  "Lepanto": "biomass"
+  "Lepanto": "biomass",
+  "Sta Fe": "unknown",
+  "Collipulli": "unknown",
+  "Totoral": "unknown",
+  "Ancali": "unknown",
+  "Nueva Renca": "gas",
+  "Laja CMPC": "biomass",
+  "Curanilahue": "coal",
+  "Cabrero": "biomass",
+  "Curacautin": "geothermal",
+  "D. Almagro": "gas",
+  "Concon": "gas",
+  "Lautaro": "biomass",
+  "Degan": "oil"
 }
 
 
-def get_xls_data(session = None):
+def get_xls_data(target_datetime = None, session = None):
     """Finds and reads .xls file from url into a pandas dataframe."""
 
-    s = session or requests.Session()
-    document_url = 'https://sic.coordinador.cl/informes-y-documentos/fichas/operacion-real/'
-    req = s.get(document_url)
-    soup = BeautifulSoup(req.text, 'html.parser')
-
-    # Find the latest file.
-    generation_link = soup.find("a", {"title": "Descargar archivo"})
-    extension = generation_link["href"]
-    base_url = "https://sic.coordinador.cl"
-    data_url = base_url + extension
-
-    date_pattern = r'OP(\d+)\.xls'
-    date_str = re.search(date_pattern, extension).group(1)
-
-    date_no_tz = arrow.get(date_str, "YYMMDD")
-    date = date_no_tz.replace(tzinfo='Chile/Continental')
-
+    if not target_datetime:
+        s = session or requests.Session()
+        document_url = 'https://sic.coordinador.cl/informes-y-documentos/fichas/operacion-real/'
+        req = s.get(document_url)
+        soup = BeautifulSoup(req.text, 'html.parser')
+
+        # Find the latest file.
```

Comment:
In the eventuality that their website is not maintained anymore and the html <a> tag is never udpated, one way to find the latest xls file is to call directly the excel URL "coordinador.cl/wp-content/uploads/estadisticas/operdiar/18/OP181127.xls" with the latest date returning a "status 200" request starting from date = now
One other way to find

Reply:
That's more a nice-to-have than a must have, but, i'm afraid that SIC is not maintaining this webpage anymore in favor of a newest one

Reply:
Yes right now you can use target datetime to get files after 6th November, we can change this if the website stops updating permanently.